pkg/tbtc/tbtc.go

```diff
@@ -26,6 +27,9 @@ const (
 	DefaultPreParamsGenerationConcurrency = 1
 )
 
+// TODO: Can we move it to constants?
```

Comment:
If that depends on the runtime we can't. Can we remove that TODO in that case?

Reply:
Yeah, I missed removing this comment. 5b019452b5470e12dfc18ebf11e9e6606489d19b